tweet_on_upcoming_and_recently_released_movies_using_azure_openai/main.bal

```diff
@@ -0,0 +1,68 @@
+import ballerina/http;
+import ballerina/io;
+import ballerinax/azure_openai.text;
+import ballerinax/themoviedb;
+import ballerinax/twitter;
+
+configurable string moviedbApiKey = ?;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+
+configurable string twitterApiKey = ?;
+configurable string twitterApiSecret = ?;
+configurable string twitterAccessToken = ?;
+configurable string twitterAccessTokenSecret = ?;
+
+const API_VERSION = "2023-03-15-preview";
+const MAX_TWEET_LENGTH = 280;
+
+// Twitter API client configuration
+twitter:ConnectionConfig twitterConfig = {
+    apiKey: twitterApiKey,
+    apiSecret: twitterApiSecret,
+    accessToken: twitterAccessToken,
+    accessTokenSecret: twitterAccessTokenSecret
+};
+
+public function main() returns error? {
+    // Get information on upcoming and recently released movies from TMDB
+    final themoviedb:Client moviedb = check new themoviedb:Client({apiKey : moviedbApiKey});
+    themoviedb:InlineResponse2001 result = check moviedb->getUpcomingMovies();
+
+    // Generate a tweet about five movies using Azure OpenAI   
+    final text:Client azureOpenAI = check new (
+        config = {httpVersion: http:HTTP_1_1, auth: {apiKey: openAIToken}},
+        serviceUrl = serviceUrl
+    );
+    string prompt = "Instruction: Generate a creative and short tweet about upcoming and recently related movies and please include the following. Movies : ";
+    int i = 0;
+    foreach var movie in result.results {
+        i = i + 1;
+        prompt += string `${i.toString()}. ${movie.title} - ${movie.overview} `;
+        if i == 5 {
```

Comment:
Define this as a const

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/63/commits/3ad5e3955599d64dd791ba6c368ca15db99a7121.